etcdserver/v3_server.go

```diff
@@ -206,6 +210,20 @@ func (s *EtcdServer) LeaseGrant(ctx context.Context, r *pb.LeaseGrantRequest) (*
 	return result.resp.(*pb.LeaseGrantResponse), nil
 }
 
+func (s *EtcdServer) LeaseLookup(ctx context.Context, r *pb.LeaseLookupRequest) (*pb.LeaseLookupResponse, error) {
+	if r.ID == int64(lease.NoLease) {
+		return nil, lease.ErrLeaseNotFound
+	}
+	result, err := s.processInternalRaftRequestOnce(ctx, pb.InternalRaftRequest{LeaseLookup: r})
```

Comment:
Will make it local. Was trying to get expiry from promote lease, but I dropped it, so we don't need to go through raft. Thanks.